core/src/main/java/brooklyn/entity/basic/BrooklynConfigKeys.java

```diff
@@ -143,6 +143,7 @@
      * component is up, but this entity does not care about the dependent component's actual config values.
      */
 
+    public static final ConfigKey<Boolean> PROVISION_LATCH = newBooleanConfigKey("provision.latch", "Latch for blocking location provision until ready");
```

Comment:
Oh wait, `SoftwareProcess` itself isn't consistent with which keys it aliases. Ignore me.